src/pushNotifications/TokenHandling.ts

```diff
@@ -44,17 +46,18 @@ const removeTokenFromServer = async (token: string) => {
       'Content-Type': 'application/json'
     },
     body: JSON.stringify({
-      token
+      notification_device: { token }
```

Comment:
this is the fix for the server communication!

Reply:
thank you for the change ☺️